eclair-core/src/main/resources/reference.conf

```
@@ -18,6 +18,8 @@ eclair {
 
   watcher-type = "bitcoind" // other *experimental* values include "electrum"
 
+  backup-file = "eclair.sqlite.backup"
```

```suggestion
  backup-file = "eclair.sqlite.bak"
```

Not sure why we need to make this configurable while other files (e.g. `eclair.sqlite`) have a fixed name.